### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by EmilyOng**
Make sure these boxes are checked before your pull request is ready to be reviewed and merged. Thanks!

* [x] all tests pass -- `rake test:all`
* [x] code is in uniquely-named feature branch, and has been rebased on top of latest master (especially if you've been asked to make additional changes)
* [x] pull request is descriptively named with #number reference back to original issue
* [x] if possible, multiple commits squashed if they're smaller changes
* [x] reviewed/confirmed/tested by another contributor or maintainer
* [x] `schema.rb.example` has been updated if any database migrations were added

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/wiki/contributing-to-public-lab-software

We have a loose schedule of reviewing and pulling in changes every Tuesday and Friday, and publishing changes on Fridays. Please alert developers on plots-dev@googlegroups.com when your request is ready or if you need assistance.

Thanks!

### CODE REVIEW

**Comment by EmilyOng** [comment]
@jywarren @ananyo2012  thanks!

**Comment by ananyo2012** [comment]
@EmilyOng This looks good. But you didn't need to make a separate PR for this. You could simply amend your commit in #1050. To amend a commit follow these steps
1. Make you changes and then add them using `git add .`
2. Amend the commit using `git commit --amend` . This will amend the previous commit and include the new changes in it. You will be prompted to edit the commit message. You can just use the same commit message and save it
3. Force push your changes using `git push -f origin branchname` where `branchname` is name of your branch

File: README.md [hunk 1]
```diff
@@ -28,190 +28,13 @@ Please read and abide by our [Code of Conduct](https://publiclab.org/conduct); o
...
-
-****
-
-## First time?
-
-New to open source/free software?, Here are a selection of issues we've made especially for first-timers. We're here to help, so just ask if one looks interesting : https://github.com/publiclab/plots2/projects/2
-
-
-We also have a slightly larger list of easy-ish but small and self contained issues: https://github.com/publiclab/plots2/labels/help-wanted
+To start off, please visit the links provided below for more information:
```
Comment (ananyo2012) [inline README.md]: Remove this line

**Comment by EmilyOng** [comment]
done

**Comment by ananyo2012** [comment]
Wait let me do the full review then you can submit all the changes.

**Comment by ananyo2012** [comment]
Make it this way

## Table of Contents

1. [Simple Installation with Cloud9](https://github.com/publiclab/plots2/wiki/Simple-Installation-for-Cloud9) 
2. [Prerequisites](https://github.com/publiclab/plots2/wiki/Prerequisites)
3. [Installation](https://github.com/publiclab/plots2/wiki/Installation)
4. [Testing](https://github.com/publiclab/plots2/wiki/Testing)
5. [API](https://github.com/publiclab/plots2/wiki/API)
6. [Bugs and Support](https://github.com/publiclab/plots2/wiki/Bugs-and-Support)

**Comment by ananyo2012** [comment]
Keep the Internationalization section in the Readme Just after the Table of Contents. Also keep the Developers and First-timers section in Readme.

**Comment by ananyo2012** [comment]
Also please add this line in the the developers section `Join our gitter chat at https://gitter.im/publiclab/publiclab`

**Comment by EmilyOng** [comment]
done

File: README.md [hunk 2]
```diff
@@ -28,144 +28,14 @@ Please read and abide by our [Code of Conduct](https://publiclab.org/conduct); o
 
 ====
 
-## Simple installation with Cloud9
+Table of Contents
```
Comment (ananyo2012) [inline README.md]: Put the header marker `##`

**Comment by EmilyOng** [comment]
yes

**Comment by ananyo2012** [comment]
@jywarren Any more changes that you would like to suggest?

**Comment by jywarren** [comment]
Hi, I'm sorry, but I disagree with moving Installation out of the README -- I think we should keep that in the README because it's important for that to show up on the front page. If you move that back in, I think this looks good. 

Thank you for your hard work on this, and apologies that there was ambiguity -- and that it's difficult for me to provide timely feedback on a weekend. We deeply appreciate your contribution!

**Comment by EmilyOng** [comment]
done @jywarren

**Comment by ananyo2012** [comment]
@EmilyOng Uh I think you didn't follow what I mentioned in https://github.com/publiclab/plots2/pull/1056#issuecomment-264692627. And now there are 5 commits for the changes. I hope you are using git in the terminal? You need to squash these commits into one so there is only a single commit. Follow these steps
1. `git rebase -i  HEAD~5`
2. This will open the rebase window. You need to replace the word `pick` with `squash` for the first 4 commits i.e. except `ad98c92` commit which is the last one. Then save it using `Ctrl+X` if the default editor is nano.
3. Then you will be prompted to enter a single commit message. Just enter the commit message and save it.
4. Force push using `git push -f origin patch-7`

**Comment by EmilyOng** [comment]
@ananyo2012

pick 6b1604d Add ability to sort wiki index alphabetically (#1049)
pick 3bf9f7d /people page explanatory text and "related question" button, fixes #991 & fixes #979 (#1041)
pick a46e991 added MathJax library (#1045)
pick a20ca3a Remove history graph for wildcard tag display (Fixes #878) (#1055)
pick c440eb0 Update README.md

**Comment by ananyo2012** [comment]
@EmilyOng Hmm this shouldn't be the last 5 commits. They are all related to the Update README.md. Are you on the right branch? What is the output of `git branch` ?

**Comment by EmilyOng** [comment]
* master @ananyo2012

**Comment by ananyo2012** [comment]
Ok you should be on the `patch-7` branch. Move to the patch-7 branch using `git checkout patch-7`

**Comment by EmilyOng** [comment]
ctrl x gives -- ^X mode (^]^D^E^F^I^K^L^N^O^Ps^U^V^Y)

**Comment by ananyo2012** [comment]
Just follow the instructions given in the terminal  carefully. After Ctrl+X you need to enter `yes` and press enter. That will save the changes.

**Comment by EmilyOng** [comment]
```
pick ad98c92 Update README.md

# Rebase a20ca3a..ad98c92 onto a20ca3a (5 command(s))
#
# Commands:
# p, pick = use commit
# r, reword = use commit, but edit the commit message
# e, edit = use commit, but stop for amending
# s, squash = use commit, but meld into previous commit
# f, fixup = like "squash", but discard this commit's log message
# x, exec = run command (the rest of the line) using shell
#
# These lines can be re-ordered; they are executed from top to bottom.
#
# If you remove a line here THAT COMMIT WILL BE LOST.
#
# However, if you remove everything, the rebase will be aborted.
#
# Note that empty commits are commented out




-- INSERT --

```

**Comment by ananyo2012** [comment]
Ok thats the last commit. Just replace the word `pick` with `squash` for the 4 commits above it. Then do Ctrl+X to save and you will prompted to enter a single commit message.

**Comment by ananyo2012** [comment]
FYI commits are those lines that don't have a `#` before it.

**Comment by EmilyOng** [comment]
whats the ctrl x key for mac?

**Comment by ananyo2012** [comment]
Hmm you will have the instruction in the editor below for save.

**Comment by ashleypt** [comment]
command-x

@EmilyOng  https://support.apple.com/en-ph/HT201236

**Comment by EmilyOng** [comment]
thanks @ashleypt @ananyo2012 but nothing happens when i click command-x

**Comment by ananyo2012** [comment]
I think there should be instruction for `Exit` below the editor window. Just follow that.

**Comment by ryzokuken** [comment]
You exit vim by doing `ESC` then `:wq` to write and quit or `:q!` to quit and remove changes

**Comment by EmilyOng** [comment]
users-MacBook-Air-2:~ Emily$ cd plots2
users-MacBook-Air-2:plots2 Emily$ ls readme.md
readme.md
users-MacBook-Air-2:plots2 Emily$ git rebase -i HEAD~5
Cannot 'squash' without a previous commit
users-MacBook-Air-2:plots2 Emily$

**Comment by ananyo2012** [comment]
You need to keep the most recent commit. You just have to squash the first 4 commits. Keep the word `pick` before the very last one.

You will probably get a message like you are middle of a rebase. If you get so abort the rebase using `git rebase --abort` and start over again.

**Comment by EmilyOng** [comment]
yes i squashed all except last

**Comment by ananyo2012** [comment]
Sorry I am traveling now. I will respond to you soon. @ryzokuken Can you see this once if you are free?

**Comment by ryzokuken** [comment]
@ananyo2012 sure. @EmilyOng what seems to be the problem?

**Comment by EmilyOng** [comment]
screen shot 2016-12-05 at 6 47 45 pm

**Comment by ryzokuken** [comment]
you're stuck on vim?

**Comment by EmilyOng** [comment]
yes. my previous codes are:

`cd plots2`
`ls readme.md`
`git rebase -i HEAD~5`

idk how to continue after, sorry

**Comment by ryzokuken** [comment]
`ESC` then type `:wq`

**Comment by EmilyOng** [comment]
`Last login: Mon Dec  5 18:30:54 on ttys000
users-MacBook-Air-2:~ Emily$ cd plots2
users-MacBook-Air-2:plots2 Emily$ ls readme.md
readme.md
users-MacBook-Air-2:plots2 Emily$ git rebase -i HEAD~5
Cannot 'squash' without a previous commit
users-MacBook-Air-2:plots2 Emily$ `

**Comment by ryzokuken** [comment]
now you're out of vim

**Comment by EmilyOng** [comment]
yes

**Comment by EmilyOng** [comment]
yes squash all except pick ad98c92

**Comment by ryzokuken** [comment]
you're out of the rebase, right?

**Comment by EmilyOng** [comment]
I'm at plots2-bash

**Comment by ryzokuken** [comment]
you don't necessarily need to rebase in order to squash those commits

**Comment by EmilyOng** [comment]
what do i do?

**Comment by ryzokuken** [comment]
there's also an easy way.
try doing 
`git reset --soft HEAD~5 && git commit -m "Update README.md"`

**Comment by ryzokuken** [comment]
Don't forget do use `-f` when pushing to `origin` though.

**Comment by EmilyOng** [comment]
@ryzokuken thanks!! 😄

**Comment by ryzokuken** [comment]
@EmilyOng don't mention it. Also, try doing things the lazy way, it helps.

**Comment by ryzokuken** [comment]
@ananyo2012 the default editor is not `nano`, but `vim` :P That was what was causing issues.

**Comment by ananyo2012** [comment]
Uh yes thanks there @ryzokuken. This is ready for merge now @jywarren

**Comment by EmilyOng** [comment]
thanks

**Comment by jywarren** [comment]
Hello all and wow, great teamwork -- and great persistence, @EmilyOng ! I actually also have `vim` as my default editor.

And @ryzokuken -- i didn't know about `git reset --soft HEAD~5` -- cool! 

For future reference, Git more recently allows me to `squash and merge` from the PR interface -- so squashing is not quite as important in a PR -- but it's a really great thing to know how to do, and I think the interactive `git rebase -i` is super useful. 

Merging, thank you very much! 🎉 😄

**Comment by jywarren** [comment]
Thank you once more for your patience and persistence! Closed and accepted, and we'd love to have another chance to work with you!

**Comment by EmilyOng** [comment]
thanks!

**Pull request merged**